e2e/nomostest/nt.go

```diff
@@ -711,6 +714,42 @@ func (nt *NT) portForwardPrometheus() {
 		prometheusServerDeploymentName,
 		nt.prometheusPortForwarder,
 	)
+
+	if err := nt.emptyPrometheusCache(); err != nil {
+		nt.T.Fatalf("failed to empty prometheus cache: %v", err)
+	}
+}
+
+// emptyPrometheusCache empty the cache in Prometheus.
+func (nt *NT) emptyPrometheusCache() error {
+	nt.T.Helper()
+	ctx, cancel := context.WithCancel(nt.Context)
+	defer cancel()
+
+	duration, err := retry.Retry(nt.DefaultWaitTimeout, func() error {
+		port, err := nt.prometheusPortForwarder.LocalPort()
+		if err != nil {
+			return err
+		}
+		c, err := prometheusapi.NewClient(prometheusapi.Config{
+			Address: fmt.Sprintf("http://localhost:%d", port),
+		})
+		if err != nil {
+			return err
+		}
+		v1api := prometheusv1.NewAPI(c)
+		// DeleteSeries only marks the time series for deletion
+		if err = v1api.DeleteSeries(ctx, nil, time.Time{}, time.Now()); err != nil {
+			return err
```

Comment:
Add extra error message context for both of these calls to the api. The errors aren't very descriptive. ex: `server_error: server error: 500`

Reply:
done